web/impact/impact/v1/views/clone_criteria_view.py

```diff
@@ -0,0 +1,40 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+from rest_framework.response import Response
+
+from accelerator.models import JudgingRound
+from impact.v1.views.impact_view import ImpactView
+from impact.v1.helpers.model_helper import (
+    READ_ONLY_ID_FIELD,
+    READ_ONLY_INTEGER_FIELD,
+    READ_ONLY_OBJECT_FIELD,
+    READ_ONLY_STRING_FIELD,
+)
+from impact.v1.helpers import (
+    CriterionHelper,
+    CriterionOptionSpecHelper,
+)
+
+ROUND_DOES_NOT_EXIST_ERROR = "Judging Round {} does not exist"
+
+
+class CloneCriteriaView(ImpactView):
+    view_name = "clone_criteria"
+    model = JudgingRound
+
+    @classmethod
+    def fields(cls):
+        return {}
+
+    def get(self, request, source_pk, target_pk):
```

Comment:
since we are creating a new resource this should be a 'post' method rather than a GET since get requests should not alter state.